src/directives/mdl.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  bind () {
-    componentHandler.upgradeElements(this.el)
+  bind (el) {
+    componentHandler.upgradeElements(el)
```

Comment:
`componentHandler.upgradeElements(this && this.el || el)`
So we can support Vue 1